### DISCUSSION TITLE
can't rewind chat with up/down arrows

### DISCUSSION THREAD

**Discussion by ticotaco72** [ppy/osu#13000] [Ideas] [answered]
up/down arrows only change volume, you can't scroll chat with them

**Comment by UselessToucan** [comment]
What about PageUp/PageDown/Home/End keys? Do they scroll the chat?

**Comment by flashwave** [comment, marked as answer]
I'd personally expect the up and down keys to pull your previous messages into the input box, like in most console window things, rather than having it scroll the view.

**Reply by Joehuu** [reply]
I believe this is the intended behavior and is recently implemented in https://github.com/ppy/osu/pull/21228.

**Comment by SuperMeepEnby** [comment]
Are you sure that your focus is in the chat window ?

**Comment by ticotaco72** [comment]
@UselessToucan Pageup/Pagedown scroll chat, but home/end don't do anything
@SuperMeepBoy  yes

**Comment by SuperMeepEnby** [comment]
Can someone reproduce the bug ? Arrows work well on my computer.

**Comment by ticotaco72** [comment]
@SuperMeepBoy have you built form source or are you using prebuilt release? maybe it's similar issue to #3541 and text box is focusing up/down buttons; i use prebuilt
